internal/node/node.bzl

```diff
@@ -179,7 +179,6 @@ def _nodejs_binary_impl(ctx):
     _write_loader_script(ctx)
 
     env_vars = "export BAZEL_TARGET=%s\n" % ctx.label
-    env_vars += "export BAZEL_WORKSPACE=%s\n" % ctx.workspace_name
```

Comment:
hmm, we didn't expect this anywhere else? looks like a breaking change to remove?

Reply:
its undocumented & internal use only, so I wouldn't consider it a breaking change